sigs.yaml

```diff
@@ -289,6 +289,33 @@ workinggroups:
         - github: zhlhahaha
           name: "Howard Zhang"
           company: ARM
+    - dir: wg-code-quality
+      name: Code-Quality Working Group
```

Comment:
Since we've decided to change the github label to `kind/cleanup`, should we align the name of the WG accordingly?

Reply:
Not sure about that - code-quality does bring a certain context where the phrase cleanup can mean anything.